releasenotes/notes/adding-bpf-network-tracer-998cb72e5a8562e9.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Adds eBPF-based network collection component called network-tracer.
```

Comment:
Since this PR adds a new process that's started by default when the Agent is installed, the release note should state that and describe what users can expect the binary to do/have access to (typically, a link to public docs would be perfect)

Reply:
I'm not sure we want to put out much documentation about this agent just yet (as the storage and frontend pieces aren't done just yet). @conorbranagan thoughts?

While its started by default, it should immediately exit with a status code of 0, as it is not enabled via: 
```
process_config:
  network_tracing_enabled:true
```

Reply:
This should be fine now since it needs to be explicitly started by the user.